src/_guides/language/analysis-options.md

```diff
@@ -181,8 +181,9 @@ include: package:pedantic/analysis_options.yaml
   When a **new version of `pedantic`** is published,
   code that previously passed analysis might **start failing analysis.**
   We recommend updating your code to work with the new rules.
-  Other options are to [use a specific version][]
-  of `pedantic`, explicitly enable individual linter rules,
+  Other options are to use a [specific version of pedantic,][]
+  include a specific version of the [pedantic analysis options file,][]
```

Comment:
We want to give one suggestion here, not two.

The original idea was that they would depend on a specific version of `pedantic`, but we realized this would be horribly broken as soon as you want to depend on two packages that are using different versions of `pedantic`--it would be a version conflict. So actually we need everyone on the same (or at least, compatible) version of `pedantic`.

The new form of words is better though, we want people to use a specific version of the options, not of the package. So e..

Other options are to include a specific version of the [pedantic analysis options file,][], explicitly enable ...